pkg/signals/signal_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
this is causing codegen CI to fail..can you undo change to this file

Reply:
Not sure where this came in. Removed

Reply:
Figured it out: the `make codegen` created this change. I removed if